plugins/org.preesm.codegen.xtend/xtend-src/org/preesm/codegen/xtend/printer/c/CHardwarePrinter.xtend

```diff
@@ -278,7 +278,7 @@ class CHardwarePrinter extends CPrinter {
 			}
 			«IF this.usingPapify == 1»
 				#ifdef _PREESM_PAPIFY_MONITOR
-				event_destroy();
+				//event_destroy();
```

Comment:
please remove completely the code. Also, if this function call is removed, the #ifdef arround can be removed too.

Reply:
this line is commented because is under investigation: the function `event_destroy()` of the `eventLib.h`-library generates a segmentation fault. The problem is the management of the `PE_id` assigned by the PAPIFY code within PREESM. I should fix the known issue this week with the help @dmadronal. A debug of `eventLib.c` may be necessary to understand what causes the segmentation fault (I know which line of code causes it) and to understand how we want to solve it.  

@antoine-morvan, because of this, please let me keep this modification and let me open an issue.

Reply:
Ok, I'll stall the merge of this PR until then.

Reply:
The code was removed as suggested so the merge can go on.

Reply:
What I suggested was to wait until the bug with the eventLib is resolved, not simply removed the event_destroy call, that will actually lead to memory leaks.